.github/workflows/pythonpackage.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.6, 3.7, 3.8]
-        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Windows support is removed because upstream contains a submodule that uses a colon in path which makes git unable to clone the submodule in Windows. 
https://github.com/ChihChengLiang/milagro_bls_binding/pull/13/checks?check_run_id=691730536